src/model/PHProbe.cpp

```diff
@@ -124,6 +126,7 @@ void PHProbe::setTemperatureCompensation(float temperature) {
 }
 
 void PHProbe::setHighpointCalibration(float highpoint) {
+  // TODO: the next two lines were commented out in another branch
```

Comment:
Perhaps it is because those two lines will be executed with the midpoint calibration anyway? I think the midpoint calibration is the one that is always performed whether it is a one-point, two-point, or three-point calibration. Although redundant, it may be ok to leave them here so long as they do not cause extra alerts to the remote log.

Reply:
Agreed. I'll also note that the midpoint might not be reached if the user cancels after a high or low without completing the midpoint.